ci/kind/antrea-kind.yml

```diff
@@ -3052,7 +3052,7 @@ data:
     #  IPsecCertAuth: false
 
     # Enable managing ExternalNode for unmanaged VM/BM.
-    #  ExternalNode: false
+      ExternalNode: true
```

Comment:
For kind its not required

Reply:
We checking the antrea AKS/EKS yaml file by enabling the externalnode feature?

Reply:
Removed it for kind.